tedge/src/cli/connect/bridge_config_c8y.rs

```diff
@@ -50,10 +50,10 @@ impl From<BridgeConfigC8yParams> for BridgeConfig {
                 r#"s/dt in 2 c8y/ """#.into(),
                 r#"s/ut/# out 2 c8y/ """#.into(),
                 // Static templates
-                r#"s/us out 2 c8y/ """#.into(),
-                r#"t/us out 2 c8y/ """#.into(),
-                r#"q/us out 2 c8y/ """#.into(),
-                r#"c/us out 2 c8y/ """#.into(),
+                r#"s/us/# out 2 c8y/ """#.into(),
+                r#"t/us/# out 2 c8y/ """#.into(),
+                r#"q/us/# out 2 c8y/ """#.into(),
+                r#"c/us/# out 2 c8y/ """#.into(),
```

Comment:
The parent device still needs to publish `s/us`, `t/us`, `q/us`, and `c/us`. I think if we configure like this, we will miss the bridge for those topics for the parent device, won't we?
So, I think we still need to keep the configuration without `/#` for those topics.

Reply:
The pattern `s/us/#` match both `s/us` and `s/us/test123`.

[The multi-level wildcard represents the parent and any number of child levels](http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc385349376)

Reply:
It's written in the specs! I didn't know that. Thank you!